### ISSUE TITLE
Package Manager Console not working with path containing grave accent (`)

### ISSUE THREAD

**Issue by DVMirchev** [NuGet/Home#5072]
## Details about Problem
Package Manager Console

NuGet version: 4.1.0

VS version: MS VS Community 2017 15.1 (26403.7)

OS version:  win10 v1607 (14393.1066)

1. I tried to integrate VCPKG to a project with the command line provided by VCPKG (I like to organize my folders with a grave accent (`) in front of them, it is a valid path):

**Install-Package vcpkg.F.vcpkg -Source "F:/`vcpkg/scripts/buildsystems"**

2. Package Manager Console returned an error:

**Install-Package : Unable to find package 'vcpkg.F.vcpkg' at source 'F:/cpkg/scripts/buildsystems'.**

3. It appears that it merges ' with v to produce vertical tab which messes up the paths. 

4. Pasting the error in Notepad++ results in:

> F:/**VT**cpkg/scripts/buildsystems

From which I assumed it is converted to a verticle tab

Logs:

```
PM> Install-Package vcpkg.F.vcpkg -Source "F:/`vcpkg/scripts/buildsystems"
Install-Package : Unable to find package 'vcpkg.F.vcpkg' at source 'F:/cpkg/scripts/buildsystems'.
At line:1 char:1
+ Install-Package vcpkg.F.vcpkg -Source "F:/`vcpkg/scripts/buildsystems ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand
 
Time Elapsed: 00:00:00.0208434
PM>
```

**Comment by emgarten** [comment]
The back tick is the escape character in powershell.

does it work if you use: ``Install-Package vcpkg.F.vcpkg -Source "F:/\`vcpkg/scripts/buildsystems"`` or escape it in some other way?

**Comment by DVMirchev** [comment]
That is unfortunate.

Same result:

```
PM> Install-Package vcpkg.F.vcpkg -Source "F:/\`vcpkg/scripts/buildsystems"
Install-Package : Unable to find package 'vcpkg.F.vcpkg' at source 'F:/\cpkg/scripts/buildsystems'.
At line:1 char:1
+ Install-Package vcpkg.F.vcpkg -Source "F:/\`vcpkg/scripts/buildsystem ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand
 
Time Elapsed: 00:00:00.0863047
PM> 
```

**Comment by emgarten** [comment]
I was off on the escaping, double back tick works.

```
PM> Write-Host "F:/\`vcpkg/scripts/buildsystems"
F:/\♂cpkg/scripts/buildsystems
PM> Write-Host "F:/``vcpkg/scripts/buildsystems"
F:/`vcpkg/scripts/buildsystems
```

**Comment by DVMirchev** [comment]
Indeed. Thanks a lot for the help.

What do you think - should I file a bug in VCPKG bug-base about this. Since I just copied and pasted this from vcpkg output:

```
F:\`vcpkg\scripts\buildsystems>vcpkg.exe integrate project
Created nupkg: F:\`vcpkg\scripts\buildsystems\vcpkg.D.Development.VCPKG.vcpkg.1.0.0.nupkg

With a project open, go to Tools->NuGet Package Manager->Package Manager Console and paste:
    Install-Package vcpkg.D.Development.VCPKG.vcpkg -Source "F:/`vcpkg/scripts/buildsystems"
```

**Comment by emgarten** [comment]
@DVMirchev if that is what the instructions for VCPKG say, then yes it looks like an issue with the docs.

From what I can tell this looks like normal powershell behavior, NuGet isn't getting a chance to handle this.

**Comment by ras0219-msft** [comment]
This was a bug in vcpkg and should be fixed by Microsoft/vcpkg#964. Thanks!
